lib/Api/RingbaAccountApi.php

```diff
@@ -17,7 +17,7 @@
  * OpenAPI spec version: v1
  * 
  * Generated by: https://github.com/swagger-api/swagger-codegen.git
- * Swagger Codegen version: 2.4.0-SNAPSHOT
+ * Swagger Codegen version: 2.4.10
  */
```

Comment:
Code changes has the array/stdclass changes as well as some new API endpoint updates.